### PR TITLE
added artemis broker security related templates 2.18.0 and 7.9.0

### PR DESCRIPTION
**Pull request by gaohoward**
Fixes #

## Proposed Changes (What, How, and Why)

  -
  -
  -

### CODE REVIEW

File: src/yacfg/profiles/artemis/2.18.0/showcase.yaml.jinja2 [hunk 1]
```diff
@@ -0,0 +1,715 @@
...
+
+# cluster related configurations
+  connectors:
+    - name: artemis
+      protocol: tcp
+      address: 10.0.0.1
+      port: 61616
+    - name: test2
+      protocol: tcp
+      address: 172.18.0.1
```
Comment (jiridanek) [inline src/yacfg/profiles/artemis/2.18.0/showcase.yaml.jinja2:230]: This used to be `172.17.0.1` in artemis 2.17.0 profile. Is that change intentional?

**Review by jiridanek — APPROVED** [review]
One suspicious change from 2.17.0 profile; otherwise looks ok at a glance.

File: src/yacfg/profiles/artemis/2.18.0/showcase.yaml.jinja2 (hunk 1 above)
Comment (gaohoward) [reply src/yacfg/profiles/artemis/2.18.0/showcase.yaml.jinja2:230]: Thank @jiridanek. You have eagle eyes! :) The change is unintentional. I use search replace after copy the profile:
/2.17.0/2.18.0/g
so 17**2.17.0**.1 becomes 17**2.18.0**.1
I'll correct it. Thanks again!

**Review by enkeys — CHANGES_REQUESTED** [review]
Only two examples of "shuffled" includes,  affected is 36 files. 

I've already fixed by https://github.com/gaohoward/YamlConfiger/pull/11 

So please rebase this PR to this branch, then we can merge this one.

**Review by enkeys — APPROVED** [review]
